### DISCUSSION TITLE
freac window elements

### DISCUSSION THREAD

**Discussion by Scouda** [enzo1982/freac#308] [Help & Support] [answered]
Is there an illustration showing & explaining the freac window elements? Specifically the function of the 3 boxes at the bottom right of the main GUI? I searched the forums but could not find anything. (BTW, wonderful program, thank you!)

**Comment by enzo1982** [comment, marked as answer]
No, there's no document explaining the UI elements. The 3 boxes show the total duration of the _selected_, _unselected_ and _all_ tracks in the joblist. _Selected_ here means that the checkbox in front of the respective track entry is marked.

**Reply by Scouda** [reply]
Got it, thanks! I'll mark this answered.
